src/modules/robot/Block.h

```diff
@@ -41,14 +42,14 @@ class Block {
 
         std::vector<Gcode> gcodes;
 
-        unsigned int   steps[3];           // Number of steps for each axis for this block
+        std::array<int, k_max_actuators>   steps; // Number of steps for each axis for this block
```

Comment:
This array will be 12 bytes bigger for every block in the queue, that is 384 bytes. This is why the default needs to be 3 not 6, unless someone actually needs 6 motors.